test/test-copyright.js

```diff
@@ -83,16 +83,15 @@ test('header updating', function(t) {
     return copyright.ensure(fakeFile, mit).then(function(contents) {
       var latest = fs.readFileSync(fakeFile, 'utf8');
       t.match(contents, fakeJS, 'should contain original content');
+      t.notStrictEqual(contents, fakeJS, 'should modify content');
       t.equal(contents, latest, 'should update the file as well as return it');
     });
   });
   t.test('when header already exists', function(t) {
     var alreadySet = fs.readFileSync(fakeFile, 'utf8');
     return copyright.ensure(fakeFile, mit).then(function(contents) {
       var latest = fs.readFileSync(fakeFile, 'utf8');
-      t.match(contents, fakeJS, 'should contain original content');
-      t.equal(contents, latest, 'should update the file as well as return it');
-      t.equal(contents, alreadySet, 'should be idempotent');
+      t.equal(latest, alreadySet, 'should be idempotent');
```

Comment:
They are covered by the previous test block, so any failure introduced by code would just trigger multiple failures for no added benefit. If the tests needed to be updated because the behaviour was intentionally changed, it would have to be changed in multiple places. As a bonus, the repeated assertions also involved disk i/o.